src/actions/guides/frontend/rendering_html.cr

```diff
@@ -579,6 +579,25 @@ class Guides::Frontend::RenderingHtml < GuideAction
     # => "about a month"
     ```
 
+    ### Cycle values
+
+    The most common case is alternating an HTML class name between rows of data. Lucky comes with a `cycle` method that makes this much
+    easier to do.
+
+    ```crystal
+    posts.each do |post|
+      tr class: cycle(["bg-gray-600", ""], "") do
+        td post.title
+      end
+    end
+    ```
+
+    In this example, the first row, and all even rows, will be `<tr class="bg-gray-600">`, but the next row, and all odd rows, will be
+    `<tr class="">`. You can pass as many values as you'd like to cycle through on each iteration.
+
+    The second argument to the `cycle` method is the default value you would like to start with. If no value is given, the string `"default"`
```

Comment:
I don't understand this at all. Maybe you could combine all the explanations and then update the code example to also show what it translates into?

Reply:
I actually don't understand how that works either. The whole cycle system is a bit complex. My guess is it's something like this (untested)

```crystal
cycle([1, :cheese, "something", 42])

1 - "default"
3 - "1"
4 - "cheese"
5 - "something"
6 - "42"
```

because the "default" value for that second argument is `"default"`. It's kind of confusing though, and in the example of alternating classes, I don't think I've ever used a classname of "default" before 😕  Maybe I just leave that off, and add "default" in to the example?

Reply:
ok, actually.. I just threw it in to a fresh new app to test, and I have no clue what that second argument does. It didn't seem to matter what I put in it... So I'm just going to leave it off and pretend it doesn't exist for now 😂